### PR TITLE
Move `cmd_interpreter` test into `not-windows-any` + diff btw `cmd_i` tests of `linux-x64` and `darwin-x64`

### PR DESCRIPTION
**Pull request by kazarmy**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [X] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [X] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

This pr reduces duplication by moving more tests from `archos`' `linux-x64`/`darwin-x64` into `not-windows-any`.

**Test plan**

All builds are green. Tests are run in `archos/not-windows-any` as appropriate.

**Closing issues**

...

### CODE REVIEW

File: test/db/archos/darwin-x64/cmd_i [hunk 1]
```diff
@@ -7,7 +7,7 @@ e io.cache=true
 "(show_fname fname; wz $0; wtf `prz`; o `prz`; i~file; ij; rm `prz`)"
 .(show_fname A\x1b¢\302\200€𝄞\363\240\201\201\\.bin)
 .(show_fname B\x1b¢\302\200€𝄞\363\240\201\201\\.bin)
-# INVALID FILENAME .(show_fname B\x1b¢\302\200\200€𝄞\363\240\201\201\\.bin)
+# INVALID FILENAME .(show_fname B\x1b¢\302\200\201€𝄞\363\240\201\201\\.bin)
```
Comment (XVilka) [inline test/db/archos/darwin-x64/cmd_i:10]: I wonder why this change?

File: test/db/archos/linux-x64/cmd_i [hunk 2]
```diff
@@ -7,12 +7,14 @@ e io.cache=true
 "(show_fname fname; wz $0; wtf `prz`; o `prz`; i~file; ij; rm `prz`)"
 .(show_fname A\x1b¢\302\200€𝄞\363\240\201\201\\.bin)
 .(show_fname B\x1b¢\302\200€𝄞\363\240\201\201\\.bin)
-# INVALID FILENAME .(show_fname B\x1b¢\302\200\200€𝄞\363\240\201\201\\.bin)
+.(show_fname B\x1b¢\302\200\201€𝄞\363\240\201\201\\.bin)
```
Comment (XVilka) [inline test/db/archos/linux-x64/cmd_i:11]: Same here

Comment (kazarmy) [reply test/db/archos/linux-x64/cmd_i:11]: Oh I intend to fix the JSON after this pr, and it's easier to see whether the fix works if the highlighted byte is different from the previous byte ... I'm making the change now just in case I forget later

File: test/db/archos/darwin-x64/cmd_i (hunk 1 above)
Comment (kazarmy) [reply test/db/archos/darwin-x64/cmd_i:10]: This is just so that `archos/linux-x64/cmd_i` and `archos/darwin-x64/cmd_i` are in sync

**Comment by wargio** [comment]
can you mark the test as broken?

**Review by XVilka — APPROVED** [review]

**Comment by kazarmy** [comment]
> can you mark the test as broken?

sure d44aaa6f36833fa32a719c2f94c35655c42c56aa

**Review by wargio — APPROVED** [review]
